Libplanet/Net/Stun/Messages/ConnectRequest.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using System.Net;
+using Libplanet.Net.Stun.Attributes;
+
+namespace Libplanet.Net.Stun.Messages
+{
+    internal class ConnectRequest : StunMessage
+    {
+        public ConnectRequest(IPEndPoint peerAddr)
```

Comment:
We'd better to have a consistent term for `IPEndPoint`.  Sometimes we name `IPEndPoint` values “address” and sometimes “end point.”